Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityOptimizeUtils.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    public class MixedRealityOptimizeUtils
+    {
+        public static void SetDepthBufferSharing(bool enableDepthBuffer)
+        {
+            PlayerSettings.VROculus.sharedDepthBuffer = enableDepthBuffer;
+
+#if UNITY_2019
+        PlayerSettings.VRWindowsMixedReality.depthBufferSharingEnabled = enableDepthBuffer;
+
+        PlayerSettings.VRWindowsMixedReality.depthBufferFormat = set16BitDepthBuffer ? 
```

Comment:
Getting an error here in 2019.1.1, `The name 'set16BitDepthBuffer' does not exist in the current context`

Reply:
Ah damn, sorry, I didn't push last commit.